alfresco-solrapi-client/alfresco-solrapi-client-spring/src/main/java/eu/xenit/alfresco/solrapi/client/spring/SolrApiSpringClient.java

```diff
@@ -186,6 +193,77 @@ public SolrTransactions getTransactions(Long fromCommitTime, Long minTxnId, Long
         return result.getBody().getNodes();
     }
 
+    @Override
+    public GetTextContentResponse getTextContent(Long nodeId, String propertyQName) {
+        UriComponentsBuilder uriBuilder = UriComponentsBuilder.fromHttpUrl(url).path("/textContent");
+        conditionalQueryParam(uriBuilder, "nodeId", nodeId, Objects::nonNull);
+        conditionalQueryParam(uriBuilder, "propertyQName", propertyQName, Objects::nonNull);
+
+        ResponseEntity<Resource> exchange = restTemplate
+                .exchange(uriBuilder.toUriString(), HttpMethod.GET, null, Resource.class);
+
+        GetTextContentResponse content = new GetTextContentResponse();
```

Comment:
this filling up of the content could be extracted in a separate function

Reply:
Ok, I did that.